packages/ui-library/src/components/forms/toggle-switch/index.css.ts

```diff
@@ -14,9 +14,9 @@ export const styleCustom = typeSafeNestedCss/* css */ `
 
 export const { tokenizedLight: toggleSwitchLight, tokenizedDark: toggleSwitchDark } = renderThemedCssStrings(
   (componentTokens, semanticTokens) => {
-    const { ToggleSwitch, FormLabel } = componentTokens.Forms;
+    const { ToggleSwitch, FormLabel } = componentTokens.cmp;
 
```

Comment:
I haven't narrowed down why this is happening just yet, but I can see that the Toggle Switch icons colour is different to what's on develop. I spotted a few other things such as the icons being vertically aligned and the knob being the incorrect colour in several states, but I think they can be resolved in a separate ticket

Reply:
as discussed, figma was behind therefor the colors differed